site/building.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Building and Developing
+menu_order: 90
+---
+
+The following topics are discussed: 
+
+ * [Developing](#developing)
+ * [Debugging](#debugging)
+
+
+## <a name="developing"></a>Developing
+
+Building Scope from source depends on the latest version of [docker](https://www.docker.com/), so please install that before proceeding.
+
+The main build is `make`, which builds the UI build container, builds the UI in said container, builds the backend build container, builds the app and probe in said container, and finally pushes the lot into a Docker image called **weaveworks/scope**.
+
+    make
+
+Then, run the local build via:
+
+    ./scope launch
+
+If needed, install the tools used for managing dependencies, managing releases, and doing coverage analysis via:
+
+    make deps
+
+
+>**Note:** The tools from `make deps` depend on a local install of
+[Go](https://golang.org).
+
+## <a name="debugging"></a>Debugging
+
+Scope has a collection of built-in debugging tools to aid Scope developers.
+
+
+To get debug information in the logs launch Scope with `--debug`:
+
+    scope launch --debug
```

Comment:
I've put four spaces in for code formatting.  The back ticks in WP are rendered with a shadowed single line, whereas the spaces will put a nice shaded box around the code.